api_FTL.php

```diff
@@ -32,7 +32,7 @@
     }
 }
 
-if (isset($_GET['summary']) || isset($_GET['summaryRaw']) || !count($_GET)) {
+if (isset($_GET['summary']) || isset($_GET['summaryRaw']) || !count($_GET) && $auth) {
```

Comment:
I don't think this is the intended behavior:

Here, you only need `$auth` to be `true` if **all** other values were `false`.
If **any** other value is `true`, you don't need to be authenticated.

Example:
```php
(true || false || false && false) == true
```

Reply:
Sure, I'll add `(...)`